app/src/main/java/com/oracle/javafx/scenebuilder/app/DocumentWatchingController.java

```diff
@@ -58,49 +58,31 @@ public class DocumentWatchingController implements FileWatcher.Delegate {
     private final SceneStyleSheetMenuController sceneStyleSheetMenuController;
     private final FileWatcher fileWatcher 
             = new FileWatcher(2000 /* ms */, this, DocumentWindowController.class.getSimpleName());
-    
-    
+
     public DocumentWatchingController(DocumentWindowController documentWindowController) {
         this.documentWindowController = documentWindowController;
         this.editorController = documentWindowController.getEditorController();
         this.resourceController = documentWindowController.getResourceController();
         this.sceneStyleSheetMenuController = documentWindowController.getSceneStyleSheetMenuController();
         
-        this.editorController.sceneStyleSheetProperty().addListener(
-                (ChangeListener<ObservableList<File>>) (ov, t,
-                        t1) -> update());
+        this.editorController.sceneStyleSheetProperty().addListener((ov, t, t1) -> update());
     }
     
     public void start() {
         fileWatcher.start();
     }
-
     
     public void stop() {
         fileWatcher.stop();
     }
-    
-    
+
     public void update() {
-        /*
-         * The file watcher associated to this document window controller watches:
-         *  1) the file holding the FXML document (if any)
-         *  2) the resource file set in the Preview menu (if any)
-         *  3) the style sheets files set in the Preview menu (if any)
-         */
-        
-        final List<Path> targets = new ArrayList<>();
+        List<Path> targets = new ArrayList<>();
```

Comment:
does removing `final` has any advantage?

Reply:
Well, one could also ask, why declare variables as final? For me, getting rid of superfluous declarations is okay. Well, I personally consider this as superfluous here. Another aspect could be just to avoid reassignemnts later somewhere, this would not work with final.

But eventually, the lesser is written, the easier to read.

Reply:
I am not against `final` per se, but as Oliver says, my reasoning was that the scope of `targets` is trivial and the use of `final` is a bit of an overkill.